composer.json

```diff
@@ -36,7 +36,8 @@
         "laminas/laminas-text": "^2.9",
         "swagger-api/swagger-ui": "^4.14",
         "zircote/swagger-php": "^4.4",
-        "psr/simple-cache": "^1.0"
+        "psr/simple-cache": "^1.0",
+        "google/recaptcha": "1.2"
```

Comment:
- require.google/recaptcha : exact version constraints (1.2) should be avoided if the package follows semantic versioning